assignments/mgmt-groups/mg-HMCTS/assign.purview.json

```diff
@@ -4,7 +4,7 @@
             "assignedBy": "github.com/hmcts/azure-policy"
```

Comment:
For some reasin git thinks I've renamed the dev subscription purview assignment rather than deleting it and creating a new assignment for the mgmt group